docs/book/start/development.md

```diff
@@ -20,7 +20,7 @@ cargo install bpf-linker
 ```
 
 If you are running **macos, or linux on any other architecture**, you need to
-install LLVM 16 first (for example, with `brew install llvm`),
+install the current LLVM first (for example, with `brew install llvm`),
```

Comment:
How about just "LLVM"?

Reply:
> How about just "LLVM"?

I think then it wouldn't be clear that not any version of LLVM will work but one needs the specific / current one.

Reply:
In that case we should say 17.

Reply:
Or just "the newest stable version". Like, literally, whatever is the latest stable on apt.llvm.org at the moment. And users of non-Debian/Ubuntu distros will understand and figure out what to do. :)

Rust nightly is always switching to new LLVM already when the first rc-s are available. And we are planning to do binstalls anyway. So I don't think that manually editing the doc just to bump the version number is worth an effort.

Reply:
changed to `the newest stable version` and in `crosscompile.md` added a mention of and link to the [llvm-sys](https://crates.io/crates/llvm-sys) crate and its major version